deploy/server.js

```diff
@@ -69,7 +71,59 @@ if (process.env['DATA_SOURCE'] !== 'mock') {
   });
 }
 
-app.get('*', (req, res) => {
+let clusterApiProxyOptions = {
+  target: virtMeta.clusterApi,
```

Comment:
For these targets, we should have conditions where in prod we use the internal endpoints described in https://github.com/konveyor/virt-ui/issues/152#issuecomment-726640172 instead of the values from `virtMeta`.

e.g.

```ts
let clusterApiProxyOptions = {
  target:
    process.env['NODE_ENV'] !== 'development'
      ? 'https://api.openshift-apiserver.svc.cluster.local/'
      : virtMeta.clusterApi,
  [...]
};
```

Reply:
Now that I realize that's the endpoint for the server it all fall in place.